### DISCUSSION TITLE
Daily 24/10

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#226] [General]
El fin de semana estuve haciendo el trabajo de Wordpress y estudiando para el exámen de GP. Participé de la daily semanal, estuve haciendo tareas del repositorio de queries y js.
Hoy voy a estar estar estudiando Python y para el examen de Full del fin de semana. Más tarde estaré con la tarea de Python del repositorio

**Comment by Brian-Almada** [comment]
El fin de semana repasé GP para hacer el examen. Hoy estaré viendo cómo realizar las tareas del repositorio que me asigné. Quiero sumar también las tareas de ingles que me faltan para ponerme al día.

**Comment by rpgrca** [comment]
Hoy estoy en reunion en el trabajo asi que voy a hacer entre poco y nada. Ayer termine de armar el controlador de subastas tuve que crear algunos DTOs de articulos y lotes pero no les puse demasiada informacion, todavia se pueden agregar. Hoy a la noche y maniana sigo con el resto de los controladores.
